authorization/src/main/kotlin/com/ritense/authorization/deployment/PermissionDeployer.kt

```diff
@@ -54,10 +57,23 @@ class PermissionDeployer(
     }
 
     fun deploy(permissions: List<PermissionDto>) {
-        permissionRepository.saveAll(permissions.map { it.toPermission(roleRepository) })
+        val permissionsToSave = permissions.filter {
```

Comment:
I think when one permission can't be deployed for some reason, the application should just not boot up. This is about security after all, so I'd much rather not miss out on any permissions.

Reply:
Changed it